src/ProjectManager.java

```diff
@@ -0,0 +1,56 @@
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.util.Scanner;
+
+import org.json.simple.JSONObject;
+
+public class ProjectManager extends Employee {
+	protected int pmExperience;
+	public final static String title = "PM";
```

Comment:
is this needed?

Reply:
I thought I will implement a generic class which uses this as a parameter, but it was a wrong idea :) I delete it for all.